### PR TITLE
Fix stray ' in html

### PR DESCRIPTION
**Pull request by MasterOdin**

### CODE REVIEW

File: TAGradingServer/account/admin-rubric.php [hunk 1]
```diff
@@ -527,7 +527,7 @@ function calculatePercentageTotal() {
         var ec = 0;
         $('select.points').each(function(){
             var elem = $(this).attr('name').replace('point','ec');
-            if (!$('[name="'+elem+'"]').prop('checked') == true) {
+            if (!$('[name="'+elem+'"]').is(':checked') == true) {
```
Comment (pbailie) [inline TAGradingServer/account/admin-rubric.php]: I am unclear about the reason for this change.  A quick skim of http://api.jquery.com/prop/ suggests that either statement would be fine.  Although this test (https://jsperf.com/prop-vs-ischecked/5) suggests that `.prop('checked')` processes faster (at least on Mac OS X browsers Safari, Chrome, and Firefox).

Comment (MasterOdin) [inline TAGradingServer/account/admin-rubric.php]: Here's a jsfiddle that shows an example of why I'm doing this: https://jsfiddle.net/g850h7mh/

Basically, a disabled checkbox doesn't send a value when the form is submitted to the php page, so I use a hidden value to take it's place. Thus $('[name="ec-0-2"]') selects the hidden field which returns false for ```.prop('checked')``` but true for ```.is(':checked')```

Comment (pbailie) [inline TAGradingServer/account/admin-rubric.php]: Sounds good to me.

**Comment by pbailie** [comment]
:+1:

**Pull request merged**